jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcCrudRepository.java

```diff
@@ -388,6 +390,14 @@ protected Flux<T> findAllAsync(Condition defaultFilter) {
             }), FluxSink.OverflowStrategy.BUFFER);
     }
 
+    public Flux<T> findAsync(String tenantId, @Nullable List<QueryFilter> filters, @Nullable String dateColumn, QueryFilter.Resource resource) {
```

Comment:
The CrudRepository was not intended to have too much methods in it, but only generic methods.
I think this is too specific and should not be added their, anyway, it's just a small wrapper over existing methods.

So please, avoid adding new methods in it or it would becomes in a few month an un-readable bag of specific methods.

Anyway, it should work with null conditions or you can create a condition that resolves to `noCondition` when your filter is null, this is how I did for other use cases